srv/salt/_modules/rgw.py

```diff
@@ -230,6 +230,11 @@ def endpoints(cluster='ceph'):
 
 
 def s3connect(user):
+    """
+    Return an S3 connection
+    """
+    if access_key(user) is None or secret_key(user) is None:
```

Comment:
more pythonic:

``` python
if not smth:
  do_smth
```
You can omit the `is None` unless you specifically expect `None`. `if not` looks for `False`, `None` and empty strings.

Reply:
I am specifically looking for None here. This function (https://github.com/SUSE/DeepSea/blob/master/srv/salt/_modules/rgw.py#L146) can return None and empty strings. Empty strings are ok, but None is not.

Reply:
ok, thanks. Haven't looked that far. No objections then.